dagster_publish_mdm/resources/tailscale.py

```diff
@@ -104,3 +104,9 @@ def get(self, path: str, *args, **kwargs) -> dict:
         response = self.client().get(url=self.url(path), *args, **kwargs)
         response.raise_for_status()
         return response.json()
+
+    def delete(self, path: str, *args, **kwargs) -> dict:
+        """Make a DELETE request to the Tailscale API."""
+        response = self.client().delete(url=self.url(path), *args, **kwargs)
```

Comment:
I think we should update the docs (`docs/src/local-development/dagster.rst`) to generate a `devices:core` scope to support this